proto/server/configEvtHandler.go

```diff
@@ -402,6 +396,36 @@ func getDeletedImsisList(group, prevGroup *configmodels.DeviceGroups) (dimsis []
 	return
 }
 
+func getProvisionedSubscribers() []string {
+	rawProvisionedSubscribers, errGetMany := dbadapter.AuthDBClient.RestfulAPIGetMany(authSubsDataColl, nil)
+	if errGetMany != nil {
+		logger.DbLog.Warnln(errGetMany)
+	}
+	var provisionedSubscribers []string
+	for _, rawProvisionedSubscriber := range rawProvisionedSubscribers {
+		ueId, ok := rawProvisionedSubscriber["ueId"].(string)
+		if !ok {
+			logger.DbLog.Warnf("cannot retrieve ueId for subscriber: %v", rawProvisionedSubscriber)
+		}
```

Comment:
```suggestion
		if !ok {
			logger.DbLog.Warnf("cannot retrieve ueId for subscriber: %v", rawProvisionedSubscriber)
			continue
		}
```
If we can not get a proper ueId, we can skip it.